src/schemes/fluid/surface_tension.jl

```diff
@@ -39,10 +39,76 @@ struct SurfaceTensionAkinci{ELTYPE} <: AkinciTypeSurfaceTension
     end
 end
 
+@doc raw"""
+    SurfaceTensionMorris(surface_tension_coefficient=1.0)
+
+This model implements the surface tension approach described by [Morris2000](@cite).
```

Comment:
```suggestion
This model implements the surface tension approach described by Morris [Morris2000](@cite).
```
This here is weird as well:
<img width="991" alt="grafik" src="https://github.com/user-attachments/assets/bf6f6aa6-235c-4e7a-abe3-647a252f6f3c" />